### DISCUSSION TITLE
`hcrystaball_model`

### DISCUSSION THREAD

**Discussion by ohenry-proda** [microsoft/FLAML#839] [Q&A]
In model.py. Is it meant to be spelled like that? Or should it be `hcrystalball_model`?

**Comment by int-chaos** [comment]
yes it should be `hcrystalball_model` would you like to create a pr to change it?

**Reply by sonichi** [reply]
Could you create an issue and label it "good first issue"?

**Reply by int-chaos** [reply]
I am planning on fixing it in an upcoming pr if it isn't done by then. Is it still necessary to create the issue?

**Reply by sonichi** [reply]
> I am planning on fixing it in an upcoming pr if it isn't done by then. Is it still necessary to create the issue?

It's not necessary. The benefit of creating an issue is to avoid redundant work. In general, it's beneficial to have your plan trackable. Also, if you create an issue and label it as "good first issue" it increases the chance of getting picked up by new contributors.
